### ISSUE TITLE
Distribute the RACK CLI wheel as a release artifact

### ISSUE THREAD

**Issue by langston-barrett** [ge-high-assurance/RACK#213]
Now that we package the RACK CLI in a standard way (#209), we can release it in wheel format alongside the RACK-Box images. This will make it easier for other TAs to interact with running RACK servers.

**Comment by tuxji** [comment]
See #220 for a new requirement.  After uploading the RACK CLI wheel as a release asset, will it be possible to install the wheel on any platform (macOS and Windows as well as Linux)?

**Comment by langston-barrett** [comment]
> After uploading the RACK CLI wheel as a release asset, will it be possible to install the wheel on any platform (macOS and Windows as well as Linux)?

Wheels [are not always cross-platform](https://www.python.org/dev/peps/pep-0425/). I don't know already what exactly makes a given wheel platform-dependent, it might just be features that we don't use (like C modules). So we should investigate what `pip` is already doing (Does it mark our wheel as platform-independent?), and determine if that's actually correct (Does our wheel really work when installed on different machines? If not, could it?).

There's additionally the question of our dependencies. There are many ways we could handle this - distributing our `requirements.txt` file, distributing pre-built wheels for each of them, or just listing their package names and versions. Distributing wheels might be platform-specific, but is the most reproducible (because it's exactly the code we test with). There also might be legal issues with redistributing binary distributions like wheels (that might just be fixed by inclusion of some licenses somewhere in our docs).

In short, there are quite a few questions to be answered :smile:

**Comment by langston-barrett** [comment]
Oh, one other option: We could upload the RACK CLI to pypi, the Python Package Index. That would make it easily installable via `pip install`. I haven't done that before, but I'd be happy to learn how!
